src/models/userModel.go

```diff
@@ -0,0 +1,83 @@
+package models
+
+import "database/sql"
+
+var db *sql.DB
+
+type User struct {
+	Id int
```

Comment:
We have discussed, that Id should be uuid
Id should be ID  - https://github.com/golang/go/wiki/CodeReviewComments#initialisms